bin/runFile.js

```diff
@@ -29,6 +33,10 @@ module.exports = (file, observe, observeTime) => {
             global[func] = function () {
                 return realFuncs[func].apply(this, arguments);
             };
+        if (continueRepl) global['closeBrowser'] = async () => {
```

Comment:
this can be problem if user has closeBrowser in between the scripts, will try to load REPL whenever there is a `closeBrowser`

Reply:
Mostly `closeBrowser` is the last action(run in `finally` block). If the user has it in between,  there would be another `openBrowser` if the script has further actions.

Reply:
ya even if there is another openBrowser, the problem is loading multiple REPL and thus REPL becomes not useable. 
Try loading below script,
```
const { openBrowser, closeBrowser, goto } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await closeBrowser();
        await openBrowser();
        await goto("google.com");
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```